### ISSUE TITLE
New Polymetis Building Issues

### ISSUE THREAD

**Issue by AlexanderKhazatsky** [facebookresearch/fairo#1406]
Hello, just wanted to let you guys know that in the newest version of this repo this line no longer works as expected: 

# OPTIONAL: Build custom version of libfranka instead
./scripts/build_libfranka.sh <version_tag_or_commit_hash>

**Comment by AlexanderKhazatsky** [comment]
Just to check, are you guys experiencing this on your end as well?

**Comment by AlexanderKhazatsky** [comment]
Another friendly bump, as we are trying to start data collection across ~10 labs and are all blocked by this :)

**Comment by AlexanderKhazatsky** [comment]
It would be equally helpful to just know if this isn't happening for you guys and we are doing something wrong

**Comment by exhaustin** [comment]
Hi sorry for the (very) slow response. Bandwidth has been increasingly limited for Polymetis but I'll try to address as many issues as possible in due time.
I am able to build both libfranka 0.9.0 and 0.10.0, but not 0.8.0 on our setup. https://github.com/frankaemika/libfranka/tree/0.8.0 seems to be no longer supported by Franka and seems to have been broken long ago. Is that the version you were failing to build?

**Comment by AlexanderKhazatsky** [comment]
No worries at all! I totally understand:)


Hmm alright this seems like a franka issue then and not your guy's problem.

If you have very limited bandwidth, the most critical thing for me would be some way of killing the gripper server (posted in another issue recently).

That's the only blocking issue I have, everything else is working great!

**Comment by AlexanderKhazatsky** [comment]
And yes, 0.8.0 was the version I was trying to build.

**Comment by exhaustin** [comment]
Thanks you for your kind works as always :)

I vaguely remember getting 0.8.0 to work by manually debugging libfranka source code and building from there.
It might also be possible to find a commit that builds, although I would probably try to upgrade the FCI as 0.8.0 is already two versions behind the stable release.
